sol/Usdx.sol

```diff
@@ -37,12 +38,19 @@ contract USDX is ERC20, Ownable {
 	function setFeed(address _newFeed) public onlyOwner {
 		AggregatorV3Interface newFeed = AggregatorV3Interface(_newFeed);
 		require(newFeed.decimals() == FEED_DECS);
+
+		(,,,,uint80 answeredInRound) = newFeed.latestRoundData();
+		(,,,uint256 updatedAt,) = newFeed.getRoundData(answeredInRound);
+		uint256 difference = uint256(block.timestamp).sub(updatedAt);
+		require(difference < FEED_EXPIRE_TIME, "Price has been updated in last 2 days, unable to set feed.");
```

Comment:
Here, you're checking whether `newFeed` has expired, but we're trying to only allow a feed replacement when `oldFeed` expires.